cypress/e2e/clue/branch/student_tests/dataflow_tool_spec.js

```diff
@@ -330,17 +330,17 @@ context('Dataflow Tool Tile', function () {
       });
       it("verify live output types", () => {
         const dropdown = "liveOutputType";
-        const outputTypes = ["Light Bulb", "Grabber"];
+        const outputTypes = ["Light Bulb", "Grabber", "Sprinkler", "Fan", "Heat Lamp"];
         dataflowToolTile.getDropdown(nodeType, dropdown).click();
-        dataflowToolTile.getDropdownOptions(nodeType, dropdown).should("have.length", 2);
+        dataflowToolTile.getDropdownOptions(nodeType, dropdown).should("have.length", 5);
         dataflowToolTile.getDropdownOptions(nodeType, dropdown).each(($tab, index, $typeList) => {
           expect($tab.text()).to.contain(outputTypes[index]);
         });
         dataflowToolTile.getOutputNodeValueText().should("contain", "off");
         dataflowToolTile.getDropdownOptions(nodeType, dropdown).last().click();
         dataflowToolTile.getDropdownOptions(nodeType, dropdown).should("have.length", 0);
-        dataflowToolTile.getDropdown(nodeType, dropdown).contains("Grabber").should("exist");
-        dataflowToolTile.getOutputNodeValueText().should("contain", "0% closed");
+        dataflowToolTile.getDropdown(nodeType, dropdown).contains("Heat Lamp").should("exist");
+        dataflowToolTile.getOutputNodeValueText().should("contain", "off");
```

Comment:
Maybe I'm not understanding this, but it seems like you continue to check for the "Grabber" too, just to make sure that is still working.

Reply:
So, this test basically just makes sure the basic options render.  It just looks for the default first item in view, which changed from "Grabber" to "Heat Lamp".  I agree there should be more functional tests than this.